reference/auth.v1.yaml

```diff
@@ -680,6 +680,144 @@ paths:
         - sessionToken: []
       tags:
         - Internal
+  /v1/consents:
+    get:
+      summary: List Consents
+      tags:
+        - Internal
+        - Consent
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: ./auth/models/consent/consents.v1.yaml
+      operationId: ListConsents
+      description: 'Retrieves the list of all different consent types and their content. '
+      requestBody:
+        content: {}
+      parameters: []
+      x-internal: true
+    parameters: []
+  '/v1/consents/{consentType}':
+    parameters:
+      - $ref: '#/components/parameters/consentType'
+    get:
+      summary: Get Latest Consent By Type
+      tags:
+        - Consent
```

Comment:
for this and  all of these requests below we should add

security:
        - sessionToken: [] 
        
    as we have done in the rest of the docs.